hrm-service/setTestEnvVars.js

```diff
@@ -0,0 +1,12 @@
+process.env.PERMISSIONS_br = 'br';
```

Comment:
Nice! Could we start moving other 'test only' settings here rather than driving them off the 'NODE_ENV'?